joomla-projects/custom-fields
# User home

Pull Request for Issue #158.
#### Summary of Changes

Adds a new flag for the media field to point to a user home directory.
#### Testing Instructions
- Create a media field and set the home flag to yes.
- Edit an article with the media field.
- Open the media manager form that field.

The directory should be a directory with the username which is initially empty.
